lib/twitter/twits.ex

```diff
@@ -44,6 +31,7 @@ defmodule Twitter.Twits do
     Post
     |> user_posts_query(user)
     |> Repo.get!(id)
+    |> Repo.preload(:likes)
```

Comment:
As I mentioned, there are 2 ways to preload: using `Repo.preload` & `Query.preload`.
At RESTAR, we tend to favor the query version as much as possible, so that Repo is only called last on the complete query.

Reply:
ok, fixed that